db/re/item_db_usable.yml

```
@@ -47553,6 +47598,468 @@ Body:
     Script: |
       specialeffect2 EF_POTION_BERSERK;
       sc_start SC_INFINITY_DRINK,1800000,0;
+  - Id: 23484
+    AegisName: Firstaid_Box_5
+    Name: First aid Box (5)
+    Type: Usable
```

EquipLevelMin is missing for all box
```suggestion
    Type: Usable
    EquipLevelMin: 5
```